catalog/pipeline/rhtap-service-push/0.1/README.md

```diff
@@ -0,0 +1,25 @@
+# RHTAP Service Push to External Registry Pipeline
+
+Tekton pipeline to push images to an external registry.
```

Comment:
should probably mention something about the pr so it is clear how it differs from push-to-external-registry

Reply:
updated